lib/modules/platform/codecommit/index.md

```diff
@@ -5,7 +5,17 @@
     Experimental features might be changed or even removed at any time.
     Subscribe to [GitHub issue #2868](https://github.com/renovatebot/renovate/issues/2868) to be notified of any changes.
 
-## Authentication
+## Authentication as a role
+
+The IAM role does not have credentials, so you will need to set up git-credentials-helper on your environment,
```

Comment:
on the other hand, it doesn't make sense to use a ROLE on renovate docker, roles are for amazon resources.

Reply:
we also don't want to run credentials config every time we run renovate, 
it's a one time thing that should be one on the machine.

Reply:
OK, let's make sure it's documented and part of any examples we have for CodeBuild

Reply:
i did not document how to do it in code build, but i wrote the documentation link to where i found how to do it,
i have not yet done any examples for CodeBuild, because i haven't tested our renovate docker with it yet.

after we release a new docker version i will try to run a dockerized version of renovate and write an example for the users